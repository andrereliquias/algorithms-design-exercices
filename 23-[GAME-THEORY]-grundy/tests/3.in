5
749
9202
3711
4823
3030
