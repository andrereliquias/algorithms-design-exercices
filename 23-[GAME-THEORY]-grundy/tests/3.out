Teco
Tico
Teco
Teco
Tico
